charts/zabbix/README.md.gotmpl

```diff
@@ -142,6 +142,11 @@ helm uninstall zabbix -n monitoring
 
 # Breaking changes of this helm chart
 
+## Version 6.1.0
+* Removing support for non-default Kubernetes features and Custom Resource objects: `IngressRoute`, `Route`, more info: #123
+* Removing support for [karpenter](https://karpenter.sh) due to the more generalistic approach: #121
+* Adding support to deploy any arbitrary manifests together with this Helm Chart by embedding them in the `.Values.extraManifests` list (#121)
+
```

Comment:
I think that is necessary add new changes about #127 PR

As for all the other changes. It's excellent. Thanks for the huge contribution.